src/libworkbench/meson.build

```diff
@@ -0,0 +1,116 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# SPDX-FileCopyrightText: Andy Holmes <andrew.g.r.holmes@gmail.com>
```

Comment:
FYI Workbench is under GPL 3 but I'm fine with these files under LGPL - looks like they can be very useful for other non-copyleft projects

Reply:
oh right, I meant to change those and have them just say `Workbench Contributors`. There's a secondary SPDX key for particular contributors that can be used. GPL-3 is fine for an internal library I think.